src/main/java/com/eternalcode/check/updater/UpdaterNotificationController.java

```diff
@@ -0,0 +1,49 @@
+package com.eternalcode.check.updater;
+
+import com.eternalcode.check.config.implementation.PluginConfig;
+import com.eternalcode.check.notification.Notification;
+import com.eternalcode.check.notification.NotificationAnnoucer;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerJoinEvent;
+
+public class UpdaterNotificationController implements Listener {
+
+    private static final Notification NEW_VERSION_AVAILABLE = Notification.chat("<b><gradient:#8a1212:#fc6b03>EternalCheck:</gradient></b> <color:#fce303>New version of EternalCheck is available, please update!");
```

Comment:
Zrób to configurable

Reply:
Po co jak to jest tylko i wyłącznie wiadomość zwrotna dla updatow